apps/conversations/models.py

```diff
@@ -166,3 +197,29 @@ def save(self, **kwargs):
 
         # force update on `Conversation.updated_at`
         self.conversation.save()
+
+
+class BaseConversationMessageReaction(models.Model):
+    """An emoji reaction to a message in
+    a conversation.
+    """
+
+    message = fk_conversation_message(
+        related_name='emoji_reactions', required=True
+    )
+    user = fk_user(related_name='reacted_by', required=False)
+    emoji_shortcode = models.CharField(max_length=24)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        abstract = True
+
+    def __str__(self):
+        value = '%s' % emoji.emojize(self.emoji_shortcode)
+        return value
+
+    # TODO: add method to convert the emoji to shortcode
+    # @property
+    # def emoji_to_shortcode(self, emoji):
+    #     value = '%s' % emoji.demojize(emoji)
+    #     return value
```

Comment:
```suggestion
    @property
    def emoji_to_shortcode(self, s):
        value = emoji.demojize(s)
        return value
```

Was this marked as `TODO` because of name conflicts? Simple fix -- just pick a different name for the parameter.

However, what is the purpose of this function existing? How would it ever get called? Seeing that it only takes in an external argument, and does not reference any attribute of this object (e.g. `self.*`) at all, it seems like something like this is more suitable as a utility function rather than an instance method.